docs/request-aggregation-metrics.csv

```diff
@@ -0,0 +1,33 @@
+Items count,Days amount,Server → ES requrest (ms),ES search exec (ms)
```

Comment:
Consider moving this to the `apinf/docs` repository, to a folder containing research and testing related to your EMQ work.